client/signup/steps/page-picker/use-cart-for-difm.tsx

```diff
@@ -194,7 +194,11 @@ export function useCartForDIFM( {
 	// [Effect] Loads required initial data
 	useEffect( () => {
 		if ( ! difmLiteProduct || ! currencyCode ) {
-			dispatch( requestProductsList() );
+			const query = {
+				type: 'partial',
```

Comment:
Can you please explain the `partial` value passed to this param? From what I can understand, the endpoint only recognizes the `jetpack` and `domains` values. (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Snqzva%2Qcyhtvaf%2Sjcpbz%2Qovyyvat%2Scebqhpg%2Qcevpr%2Qyvfg.cuc%3Se%3Qr5nqn737%2310-og)

Reply:
The term `partial` is more for readability, if the type isn't included it defaults to `all` which isn't recognized by the endpoint.